recipes-init/openrc-inittab-busybox/openrc-inittab-busybox_1.0.bb

```diff
@@ -0,0 +1,49 @@
+SUMMARY = "BusyBox configuration for OpenRC"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COREBASE}/meta/COPYING.MIT;md5=3da9cfbcb788c80a0384361b4de20420"
+
+SRC_URI = "file://inittab.in"
+S = "${WORKDIR}"
+RPROVIDES:${PN} = "${@oe.utils.conditional('VIRTUAL-RUNTIME_init_manager', 'busybox', 'virtual/openrc-inittab', '', d)}"
+
+INHIBIT_DEFAULT_DEPS = "1"
+
+do_configure[noexec] = "1"
+do_compile[noexec] = "1"
+
+USE_VT ?= "1"
+SYSVINIT_ENABLED_GETTYS ?= "1"
+
+sbindir="${@d.getVar(bb.utils.contains('PACKAGECONFIG', 'usrmerge', 'sbindir', 'base_sbindir', d))}"
+
+do_install() {
+    install -d ${D}${sysconfdir}
+    install -m 0644 ${WORKDIR}/inittab.in ${D}${sysconfdir}/inittab
+}
+
+python update_inittab() {
+    import pathlib
+
+    dest = pathlib.Path(d.getVar("D")) / d.getVar("sysconfdir").lstrip('/') / "inittab"
+
+    with dest.open('r') as fp:
+        lines = fp.readlines()
+
+    for i, baud, dev in ((i, *x.split(';')) for i, x in enumerate(d.getVar("SERIAL_CONSOLES").split())):
+        lines.append(f"{dev}:12345:respawn:${{sbindir}}/getty {baud} - vt102")
+
+    if d.getVar("USE_VT") == "1":
+        lines.append('')
+        for vt in d.getVar("SYSVINIT_ENABLED_GETTYS").split():
+            lines.append(f"tty{vt}:12345:respawn:${{sbindir}}/getty 38400 -")
+
+    lines.append('')
+
+
+    with dest.open('a') as fp:
```

Comment:
Same comments here as in the prior commit, open as 'w', strip `\n' when reading the lines above.